src/renderer/components/properties/BoundariesProperties.js

```diff
@@ -38,6 +50,8 @@ const LineProperties = props => {
       <TextProperty label={'Additional Information'} property={'h'} properties={props.properties} onCommit={props.update} className={ classes.twoColumns }/>
       <HostilityProperty properties={props.properties} onCommit={props.update}/>
       <EchelonProperty properties={props.properties} onCommit={props.update}/>
+      <Typography component='div'>Offset</Typography>
+      <Slider value={echelonOffset} defaultValue={0.5} min={0} max={1} step={0.01} onChange={handleChange} color={'secondary'} className={ classes.twoColumns }/>
```

Comment:
Remove the ```defaultValue``` since this is to be used for non-controlled components:

```javascript
<Slider value={echelonOffset} min={0} max={1} step={0.01} onChange={handleChange} color={'secondary'} className={ classes.twoColumns }/>
```